sched/scheduler/jobStatus.go

```diff
@@ -1,83 +0,0 @@
-package scheduler
```

Comment:
So I deleted it for a few reasons:
* Status is just an API all the Status is basically an interpretation of the SagaLog.  All the app logic is Saga Based this is just wire struct info.  The code would have just had an additional layer of indirection which translates it from this object to a thrift object, which I thought was kind of wasteful and didn't gain us anything
* Having it in scheduler could potentially couple status to scheduler and we may want to separate them one day for load reasons.
* The scheduler tests actually just check SagaStatus so I think we don't need this?  The thrift stays just in the server.  If it starts creeping into scheduler we can add it back.  The beauty of git having the entire history.